The error in the merge command seems not related with this PR 

https://github.com/OCA/account-invoicing/actions/runs/11469170392/job/31915851268#step:8:726
```bash
2024-10-22 22:03:10,750 564 ERROR odoo odoo.addons.account_invoice_pricelist.tests.test_account_move_pricelist: FAIL: TestAccountMovePricelist.test_account_invoice_fixed_pricelist_without_discount_secondary_currency
Traceback (most recent call last):
  File "/__w/account-invoicing/account-invoicing/account_invoice_pricelist/tests/test_account_move_pricelist.py", line 319, in test_account_invoice_fixed_pricelist_without_discount_secondary_currency
    self.assertAlmostEqual(invoice_line.price_unit, 65.41)
AssertionError: 100.0 != 65.41 within 7 places

2024-10-22 22:03:10,958 564 ERROR odoo odoo.addons.account_invoice_pricelist.tests.test_account_move_pricelist: FAIL: TestAccountMovePricelist.test_account_invoice_pricelist_with_discount_secondary_currency
Traceback (most recent call last):
  File "/__w/account-invoicing/account-invoicing/account_invoice_pricelist/tests/test_account_move_pricelist.py", line 295, in test_account_invoice_pricelist_with_discount_secondary_currency
    self.assertAlmostEqual(invoice_line.price_unit, 58.87)
AssertionError: 90.00000000000004 != 58.87 within 7 places

2024-10-22 22:03:11,360 564 ERROR odoo odoo.addons.account_invoice_pricelist.tests.test_account_move_pricelist: FAIL: TestAccountMovePricelist.test_account_invoice_pricelist_without_discount_secondary_currency
Traceback (most recent call last):
  File "/__w/account-invoicing/account-invoicing/account_invoice_pricelist/tests/test_account_move_pricelist.py", line 303, in test_account_invoice_pricelist_without_discount_secondary_currency
    self.assertAlmostEqual(invoice_line.price_unit, 65.41)
AssertionError: 100.0 != 65.41 within 7 places
```

--- a/stock_picking_invoicing/models/__init__.py
+++ b/stock_picking_invoicing/models/__init__.py
@@ -1,8 +1,6 @@
-# Copyright (C) 2019-Today: Odoo Community Association (OCA)
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-
 from . import stock_invoice_state_mixin
 from . import account_move
 from . import stock_move
 from . import stock_picking
 from . import stock_picking_type
+from . import stock_rule

--- a/stock_picking_invoicing/models/stock_rule.py
+++ b/stock_picking_invoicing/models/stock_rule.py
@@ -1,0 +1,14 @@
+# Copyright (C) 2024-Today - Akretion (<http://www.akretion.com>).
+# @author Magno Costa <magno.costa@akretion.com.br>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class StockRule(models.Model):
+    _inherit = "stock.rule"
+
+    def _get_custom_move_fields(self):
+        fields = super()._get_custom_move_fields()
+        fields += ["invoice_state"]
+        return fields

